framework/src/st_things/Make.defs

```diff
@@ -0,0 +1,133 @@
+###########################################################################
+#
+# Copyright 2017 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+
+THINGS_LIB_DIR = $(TOPDIR)/../framework/src/st_things
+THINGS_STACK_DIR = $(TOPDIR)/../framework/src/st_things/things_stack
+IOTIVITY_RELEASE=${shell echo $(CONFIG_IOTIVITY_RELEASE_VERSION) | sed 's/"//g'}
+IOTIVITY_BASE_DIR = $(TOPDIR)/../external/iotivity/iotivity_$(IOTIVITY_RELEASE)
+
+CSRCS += st_things.c st_things_util.c st_things_request_handler.c st_things_representation.c sdkapi_logger.c 
+
+CFLAGS += -I$(TOPDIR)/../framework/include/st_things/
```

Comment:
Why don't you use common include path? It can give user confused.
Due to this line, in this module, inlcuding path is,
```
#include <aa.h>
```
but user should use different usages like below.
```
#include <st_things/aa.h>
```